app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  has_many :stories
+
```

Comment:
I am not sure if any of you have used this in the past, (I know a past group of mine has) but do think we should include a dependent destroy here for the user model? That way if the user is destroyed, the stories associated with that user are also destroyed?

Reply:
100% Super good idea. I'll add it